plugins/home/src/components/VisitList/ItemCategory.tsx

```diff
@@ -27,53 +26,17 @@ const useStyles = makeStyles(theme => ({
     margin: 0,
   },
 }));
-const maybeEntity = (visit: Visit): CompoundEntityRef | undefined => {
-  try {
-    return parseEntityRef(visit?.entityRef ?? '');
-  } catch (e) {
-    return undefined;
-  }
-};
-const getColorByIndex = (index: number) => {
-  const variants = Object.keys(colorVariants);
-  const variantIndex = index % variants.length;
-  return colorVariants[variants[variantIndex]][0];
-};
-const getChipColor = (entity: CompoundEntityRef | undefined): string => {
-  const defaultColor = getColorByIndex(0);
-  if (!entity) return defaultColor;
-
-  // IDEA: Use or replicate useAllKinds hook thus supporting all software catalog
-  //       registered kinds. See:
-  //       plugins/catalog-react/src/components/EntityKindPicker/kindFilterUtils.ts
-  //       Provide extension point to register your own color code.
-  const entityKinds = [
-    'component',
-    'template',
-    'api',
-    'group',
-    'user',
-    'resource',
-    'system',
-    'domain',
-    'location',
-  ];
-  const foundIndex = entityKinds.indexOf(
-    entity.kind.toLocaleLowerCase('en-US'),
-  );
-  return foundIndex === -1 ? defaultColor : getColorByIndex(foundIndex + 1);
-};
 
 export const ItemCategory = ({ visit }: { visit: Visit }) => {
   const classes = useStyles();
-  const entity = maybeEntity(visit);
+  const { getChipColor, getLabel } = useVisitDisplay();
```

Comment:
I didn't see anywhere in this PR where the `<VisitDisplayContext>` is provided by default. Would this result in the default home throwing errors?

(Apologies if I'm missing broader context around the Home plugin--it's been awhile since I've looked through the code)